docs/src/examples/components/Segment/Variations/SegmentExampleInverted.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react'
+import { Segment, Text } from '@stardust-ui/react'
+
+const SegmentExampleInvertedShorthand = () => (
+  <div>
+    <Segment variables={siteVars => ({ color: siteVars.brand })}>Colored segment.</Segment>
```

Comment:
would suggest to at least provide fallback color in case if `brand` won't be defined in `siteVars` for some reason

Reply:
thx, done